pop-benchmarker/src/main/java/com/microsoft/testing/popbenchmarker/MainActivity.java

```diff
@@ -0,0 +1,385 @@
+                        // Done!
+                        sLOCK.release();
+                    }
+                });
+            }
+        });
+
+        getKeyGenerationTiming(new AsyncResultCallback<String>() {
+            @Override
+            public void onDone(String result) {
+                //setText(mTv_KeyGen, result);
+                tasks.remove().run();
+            }
+        });
+    }
+
+    private void getIsHardwareIsolated(@NonNull final AsyncResultCallback<String> callback) {
+        final Timer.TimerResult<Boolean> result = Timer.execute(new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+                final KeyStore keyStore = KeyStore.getInstance(ANDROID_KEYSTORE);
+                keyStore.load(null);
+                final KeyStore.Entry entry = keyStore.getEntry(KEYSTORE_ALIAS, null);
+                PrivateKey privateKey = ((KeyStore.PrivateKeyEntry) entry).getPrivateKey();
+                return isInsideSecureHardware(privateKey);
+            }
+        });
+        mThreadMarshaller.postResult(result.mResult.toString(), callback);
+    }
+
+    private void getSigningTiming(@NonNull final AsyncResultCallback<String> callback) {
+        try {
+            // Prepare JWT with claims set
+            final JWTClaimsSet claimsSet = new JWTClaimsSet.Builder()
+                    .subject("test")
+                    .issuer("https://login.microsoftonline.com")
+                    .expirationTime(new Date(new Date().getTime() + 60 * 1000))
+                    .build();
+
+            final KeyStore keyStore = KeyStore.getInstance(ANDROID_KEYSTORE);
+            keyStore.load(null);
+
+            final KeyStore.Entry entry = keyStore.getEntry(KEYSTORE_ALIAS, null);
+            final PrivateKey privateKey = ((KeyStore.PrivateKeyEntry) entry).getPrivateKey();
+
+            final RSASSASigner signer = new RSASSASigner(privateKey);
+
+            final SignedJWT signedJWT = new SignedJWT(
+                    new JWSHeader.Builder(JWSAlgorithm.RS256)
+                            //.keyID(key.getKeyID())
+                            .build(),
+                    claimsSet
+            );
+
+            final Timer.TimerResult<Void> result = Timer.execute(new Callable<Void>() {
+                @Override
+                public Void call() throws Exception {
+                    // Compute the RSA signature
+                    signedJWT.sign(signer);
+
+                    // To serialize to compact form, produces something like
+                    // eyJhbGciOiJSUzI1NiJ9.SW4gUlNBIHdlIHRydXN0IQ.IRMQENi4nJyp4er2L
+                    // mZq3ivwoAjqa1uUkSBKFIX7ATndFF5ivnt-m8uApHO4kfIFOrW7w2Ezmlg3Qd
+                    // maXlS9DhN0nUk_hGI3amEjkKd0BWYCB8vfUbUv0XGjQip78AI4z1PrFRNidm7
+                    // -jPDm5Iq0SZnjKjCNS5Q15fokXZc8u0A
+                    final String signed = signedJWT.serialize();
+                    //
```

Comment:
nit: Remove unused comment slash

Reply:
Resolved in ab28d0e09b6c6e0550bddd52f9e678ace5b40840